Demo/AuthenticatorDemo.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,361 @@
+// !$*UTF8*$!
```

Comment:
I decided to use a dedicated project instead of making the demo app a new target in the project where we define WordPressAuthenticator to avoid noise in that project file.